tests/test.py

```diff
@@ -0,0 +1,123 @@
+"""Unit test for DB12"""
+from __future__ import absolute_import
+from __future__ import division
+import pytest
+
+#pylint: disable=relative-beyond-top-level
+from ..src.DIRACbenchmark import dirac_benchmark
```

Comment:
What about that? Doesn't it work? 

```suggestion
from DIRACbenchmark import dirac_benchmark
```

Reply:
No it doesn't it gives an import error.